yml/dp02_dc2.yaml

```diff
@@ -7749,7 +7749,7 @@ tables:
     tap:std: 1
     tap:principal: 1
     ivoa:unit:
-    datatype: char
+    datatype: string
```

Comment:
Confirmed with @iagaponenko that this is the right type, matching what's actually deployed in Qserv, `VARCHAR(512)`